lib/ReactViews/ExplorerWindow/ExplorerWindow.jsx

```diff
@@ -33,15 +34,25 @@ const ExplorerWindow = createReactClass({
 
     /* eslint-disable-next-line camelcase */
     UNSAFE_componentWillMount() {
+        const { terria, viewState } = this.props;
+
         this.props.viewState.explorerPanelAnimating = true;
 
         this._pickedFeaturesSubscription = ko.pureComputed(this.isVisible, this).subscribe(this.onVisibilityChange);
+
+        if (terria.sharedFromExplorerPanel) {
+            viewState.openAddData();
```

Comment:
we don't actually need this coz the `viewCatalogMember` function in `viewState.js` actually check if the `previewed`  item is added by user or from the default data explorer

Reply:
Thanks! This also actually prompted me to check and catalog-shareable-user-added-shares were indeed not working, so I've fixed that w/ https://github.com/TerriaJS/terriajs/pull/3278/commits/19128b5edec86e58e35c86c5d741951872c1e6d0 too